openmc/tracks.py

```diff
@@ -266,6 +266,55 @@ def plot(self):
             track.plot(ax)
         return ax
 
+    def write_tracks_to_vtk(self, filename):
+        """Creates a VTK file of the tracks
+
+        Parameters
+        ----------
+        filename : str
```

Comment:
support path-like too? I'm not sure vtk supports it so you might have to write `str(filename)`

Reply:
Thanks paul I tried it and you were right SetFileName accepts str types only